etc/parameters/field_b/default.json

```diff
@@ -1 +1,20 @@
-{}
+{
+  "image_segmenter": {
+    "vision_top": {
+      "vertical_edge_threshold": {
+        "first_half_own_half_towards_own_goal": 20.0,
+        "first_half_own_half_away_own_goal": 20.0,
+        "first_half_opponent_half_towards_own_goal": 20.0,
+        "first_half_opponent_half_away_own_goal": 20.0
+      }
+    },
+    "vision_bottom": {
+      "vertical_edge_threshold": {
+        "first_half_own_half_towards_own_goal": 32.0,
+        "first_half_own_half_away_own_goal": 32.0,
+        "first_half_opponent_half_towards_own_goal": 32.0,
+        "first_half_opponent_half_away_own_goal": 32.0
+      }
+    }
+  }
+}
```

Comment:
Isn't this the same as in the default.json?

Reply:
Yes, but this way, changes in the top-level `default.json` will not change the already tuned parameters in this location.